src/App.tsx

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
-import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
+import { Switch, Route } from 'react-router-dom';
 import Dashboard from './components/dashboard/dashboard';
-import Events from './components/events/events';
 import Staff from './components/staff/staff';
 import Authorization from './components/authorization/authorization';
 import Navbar from './components/navbar/navbar';
 import Error from './components/path_error/error';
+import Events from './components/Events/Events';
 
 const App: React.FC = () => {
     return (
-        <BrowserRouter>
+        <div>
```

Comment:
Why do you use tag div instead of browser router?

Reply:
When I have another structure, I created <Route>  component  into my <Event> component and I didn't want to duplicate <BrouserRouter> and brought up a level